### PR TITLE
Fix issues found using dennis

### PR DESCRIPTION
**Pull request by jayvdb**

### CODE REVIEW

**Comment by jayvdb** [comment]
The current list of warnings and errors found by dennis can be seen at : https://travis-ci.org/jayvdb/localized_docs/builds/167009605

If desired, I can also submit a `.travis.yml` that is green: https://travis-ci.org/jayvdb/localized_docs/builds/167012787
There are still some warnings, but often they need to be fixed in the English, or need to be ignored with a feature like https://github.com/willkg/dennis/issues/80 .

**Comment by nijel** [comment]
Thanks! Having .travis check for this is good idea, PR is welcome.

File: po/tr.po [hunk 1]
```diff
@@ -16689,7 +16689,7 @@ msgstr ""
 "Ayrıca, sorgunun içerisinde, değerler için yer tutuculara sahip "
 "olabilirsiniz. Bu, sorgu içerisine ``/*`` ve ``*/`` arasına SQL açıklamaları "
 "eklenerek yapılır. Açıklamaların içerisinde, özel dizgiler "
-"``[DEĞİŞKEN{değişken-numarası}]`` kullanılır. SQL açıklamalarından çıkarılan "
```
Comment (jayvdb) [inline po/tr.po:16692]: @nijel , should this be translated?

File: po/fr.po [hunk 2]
```diff
@@ -17032,7 +17032,7 @@ msgstr ""
 "Vous pouvez aussi avoir, à l'intérieur d'une requête, des espaces réservés "
 "pour des variables. Ceci est effectué en insérant dans la requête un "
 "commentaire SQL entre ``/*`` et ``*/``. Dans le commentaire, la chaîne "
-"spéciale ``[VARIABLE{numéro de variable}]`` est utilisée. Gardez à l'esprit "
```
Comment (jayvdb) [inline po/fr.po:17035]: @nijel , should this be translated?

**Comment by jayvdb** [comment]
@nijel, ok, I will submit a `.travis.yml` soon.

I suspect that `[VARIABLE{variable-number}]` should be translated, and I think I know how to prevent it being a `dennis` error.

**Comment by nijel** [comment]
It's fine to translate it.

**Comment by jayvdb** [comment]
Ok, I'll fiddle with dennis to allow that, and then revert those edits in this commit.
As an aside, I am using this project in an training program/competition.
You can read about the task at https://wikimedia-id.github.io/besutkode/university-sample-task-1-en.html

**Comment by nijel** [comment]
Thanks! Is the competition somehow open to the public? If so, we could promote it on our social media as well...

**Comment by jayvdb** [comment]
Open only to Indonesian uni students, and registration closes in a few days.  Thanks for your offer, but I doubt you would reach anyone who we havent already reached.

**Pull request merged**